mdtraj/reporters/basereporter.py

```diff
@@ -62,7 +62,8 @@ def backend(self):
 
     def __init__(self, file, reportInterval, coordinates=True, time=True,
                  cell=True, potentialEnergy=True, kineticEnergy=True,
-                 temperature=True, velocities=False, atomSubset=None):
+                 temperature=True, velocities=False, atomSubset=None,
+                 enforcePeriodicBox=None):
```

Comment:
Should the default value be `False` rather than `bool`?

Reply:
I "copied" this from `__init__` of  class PDBReporter in openmm's [pdbreporter.py](https://github.com/openmm/openmm/blob/master/wrappers/python/openmm/app/pdbreporter.py)
In the beginning, I also feel that the default should better be a `bool` (either `False` or `True`) instead of `None`. However, after reading openmm's source code, I found that openmm deal with this parameter [in this for loop](https://github.com/openmm/openmm/blob/e53bdc5eab600b4137abd80e544cb100d1acac68/wrappers/python/openmm/app/simulation.py#L214). It seems that the reporter is designed in such a way by openmm so it is better to use `None` as a default value. Therefor, I decided to follow the way openmm does and set the default to `None`.

Reply:
If the argument is `None` that means to apply default wrapping rules: do it for periodic systems, but not for non-periodic systems.  That's the default value for other reporters.

Reply:
According to the openmm code, the enforcePeriodicBox can take one of the three values: `None`, `False`, and `True`. When using `None`, the coordinates will be wrapped if the simulation uses the periodic boundary condition and will be unwrapped if the simulation uses the nonperiodic boundary condition. This is the behavior of the previous HDF5Reporter (Please correct me if it is not true).
So the advantage of setting `None` as the default is that it will be backward compatible. Users who download the new MDTraj don't need to change their original code.

Reply:
Got it. Could you make this more clear in the mdtraj docstring then?

Reply:
Done. Thanks for pointing that out.